modules/configuring-knative-services-domains.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * serverless/getting-started-knative-services.adoc
+
+[id="configuring-knative-services-domains_{context}"]
+= Configuring domains for Knative services
+
+The Red Hat OpenShift Serverless Operator will attempt to find the public domain that is used by the cluster and use that same domain for Knative Services. Having a domain configured makes it possible to reach services in your cluster via HTTP over the internet directly, for example, using the `curl` command.
+
+Alternatively, you can configure a different domain for Knative services or even configure a different domain for different Knative services based on labels. Once you have completed this configuration, you can create a template for the domain configuration.
+
+.Procedure
+. To change the configuration for Knative services, you must edit the `config-domain` configmap in the `knative-serving` namespace using `oc edit`.
+
+   oc edit cm config-domain -n knative-serving
+
+.. Sample configmap
+
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: config-domain
+    namespace: knative-serving
+    labels:
+      serving.knative.dev/release: devel
+  data:
+    example.com: |
+
+.. Here is an example of configuration for the domain `example.org`, which will be used for routes with the label `app` set to `example`.
+
+  data:
+    example.org: |
+  selector:
+      app: example
```

Comment:
I think the indentation for the selector is not correct here. It should be:
```
data:
  example.org: |
    selector:
      app: example
```
There can be multiple domains defined and the next domain will be at the same level as example.org.

Reply:
Thanks Martin! Should be fixed in the new commit.